frontend/javascripts/admin/admin_rest_api.js

```diff
@@ -649,6 +649,10 @@ export async function getTracingForAnnotationType(
 
   const tracing = parseProtoTracing(tracingArrayBuffer, tracingType);
   // The tracing id is not contained in the server tracing, but in the annotation content
+  if(tracing.hasOwnProperty("userBoundingBox")) {
```

Comment:
> I double-checked whether the backend sends the old user bounding box and did not succeed. I am pretty sure I am doing nothing wrong here. The tracing only includes the empty userBoundingBoxes array and no userBoundingBox.

@MichaelBuessemeyer I was able to identify the cause of the missing `userBoundingBox`. As it turns out, protobuf omits null fields. Addiotionally, there needed to be an extra get request before transitioning to the new branch because the updates get applied lazily. I'll fix that.

I built this hacky workaround for moving the old userBoundingBox into the array. Please change this, as the linter doesn't seem to like it :wink:

Reply:
So the frontend doesn't need to take care of the compatibility for old tracings that only have one `userBoundingBox`?

Reply:
Oh nevermind that question. I just realized that this is code change is  in the frontend :sweat_smile: 

I adjusted the code a bit :smile:

Reply:
If it is possible to just add the old `userBoundingBox` to the new array should make the implementation backwards compatible.